FamilyTreeTesting/Scripts/ImprovedScript/DataModel.js

```diff
@@ -16,8 +16,9 @@ function getDefaultLogicUnitData(inputId, inputGender) {
         multiInvididualText: "",
         isDead: false,
         isAdopted: false,
+        gotAdopted: false,
```

Comment:
this should be merged into one property with isAdopted, like adoptedStatus, but change later